.github/main.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+        - main
+  pull_request:
+    branches:
+        - main
+
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: environment.yml
+          environment-name: untarjs 
+          condarc: |
+            channels:
+              - https://repo.mamba.pm/emscripten-forge 
+              - conda-forge
```

Comment:
Do we actually need this?

The channel is already set in the `environment.yml` file

Reply:
Sorry, I meant the entire condarc.

conda-forge is actually needed, but it's already properly set in your `environment.yml` file

Reply:
@martinRenou done